config/index.js

```diff
@@ -33,6 +33,8 @@ config.vaccineInterval = process.env.VACCINE_INTERVAL || 864e5;
 // eslint-disable-next-line camelcase
 config.vaccineCoverageInterval = process.env.VACCINE_COVERAGE_INTERVAL || 864e5;
 // eslint-disable-next-line camelcase
+config.vaccineStateCoverageInterval = process.env.VACCINE_STATE_COVERAGE_INTERVAL || 864e5;
```

Comment:
if it's from the same source we don't really need a new config for the interval just sayin

Reply:
fair enough